salt/states/beacon.py

```diff
@@ -61,7 +65,7 @@ def present(name,
         else:
             if 'test' in __opts__ and __opts__['test']:
                 kwargs['test'] = True
-                result = __salt__['beacons.modify'](name, beacon_data)
+                result = __salt__['beacons.modify'](name, beacon_data, **kwargs)
```

Comment:
I don't think this is what we want here. We already have kwargs in the `beacon_data` list up on L55. Instead, we just need to merge the `save` option into that structure.

Reply:
This was not a change by me, but took it out now. You are right, should not be there since is already in beacon_data.